Sources/StevensonCore/Release.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Vapor
+
+extension GitHubService {
+    public struct Release {
+        public let repository: Repository
+        public let branch: String
+        public let appName: String?
+        public let version: String
+
+        /// - Parameters:
+        ///   - repository: the GHRepo this release is for, ios or android
+        ///   - branch: The GIT branch name, typically of the form `release/1.2.3` or `release/1.2.3-nhs111`
```

Comment:
Let's keep the "core" clear of mentions of any specifics of our projects, for users of the app this will be specified in the slack command hint

```suggestion
        ///   - branch: The git branch name
```